<!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]> <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]> <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8">
  <title>Natural Variation in Gene Expression Modulates the Severity of Mutant Phenotypes  &middot; HGComputational Journal Club</title>
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1">


<meta name="description" content="" />

<meta name="keywords" content="epistasis, ">


<meta property="og:title" content="Natural Variation in Gene Expression Modulates the Severity of Mutant Phenotypes  &middot; HGComputational Journal Club ">
<meta property="og:site_name" content="HGComputational Journal Club"/>
<meta property="og:url" content="https://xinhe-lab.github.io/journal-club/20180206-victoria-2015-cell/" />
<meta property="og:locale" content="en">


<meta property="og:type" content="article" />
<meta property="og:description" content=""/>
<meta property="og:article:published_time" content="2018-02-06T00:00:00Z" />
<meta property="og:article:modified_time" content="2018-02-06T00:00:00Z" />

  
    
<meta property="og:article:tag" content="epistasis">
    
  

  
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@" />
<meta name="twitter:creator" content="@" />
<meta name="twitter:title" content="Natural Variation in Gene Expression Modulates the Severity of Mutant Phenotypes" />
<meta name="twitter:description" content="" />
<meta name="twitter:url" content="https://xinhe-lab.github.io/journal-club/20180206-victoria-2015-cell/" />
<meta name="twitter:domain" content="https://xinhe-lab.github.io/journal-club/">
  

  
<script type="application/ld+json">
  {
    "@context": "http://schema.org",
    "@type": "Article",
    "headline": "Natural Variation in Gene Expression Modulates the Severity of Mutant Phenotypes",
    "author": {
      "@type": "Person",
      "name": "http://profiles.google.com/+?rel=author"
    },
    "datePublished": "2018-02-06",
    "description": "",
    "wordCount": 616
  }
</script>
  



<link rel="canonical" href="https://xinhe-lab.github.io/journal-club/20180206-victoria-2015-cell/" />
<link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://xinhe-lab.github.io/journal-club/touch-icon-144-precomposed.png">
<link rel="icon" href="https://xinhe-lab.github.io/journal-club/favicon.png">
<meta name="generator" content="Hugo 0.30.2" />

  <!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.2/html5shiv.js"></script>
<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
<![endif]-->



    <link rel="stylesheet" href="https://xinhe-lab.github.io/journal-club/css/bootswatch/paper/bootstrap.min.css">


<link rel="stylesheet" href="https://xinhe-lab.github.io/journal-club/css/font-awesome.min.css">
<link rel="stylesheet" href="https://xinhe-lab.github.io/journal-club/css/style.css">


<link rel="stylesheet" href="https://xinhe-lab.github.io/journal-club/css/custom.css">



  <link rel="stylesheet" href="https://xinhe-lab.github.io/journal-club/css/highlight/default.css">


  

<script src="https://xinhe-lab.github.io/journal-club/js/jquery.min.js"></script>
<script src="https://xinhe-lab.github.io/journal-club/js/bootstrap.min.js"></script>


<script src="https://xinhe-lab.github.io/journal-club/js/highlight.pack.js"></script>
<script src="https://xinhe-lab.github.io/journal-club/js/site.js"></script>
<script>hljs.initHighlightingOnLoad();</script>


<script>
var ENABLE_POPOVER =  false , 
EXPIRE_COOKIE =  5 , 
SHOW_MODAL_TIMEOUT =  10000 , 
MOUSE_LEAVE =  true , 
MODAL_SIZE = "", 
POST_URL = "https://zapier.com/hooks/catch/1227563/", 
SIGNUP_HEADER = "Join Our Newsletter",
HEADER_IMAGE = "//placehold.it/1000x600",
IMG_DESCRIPTION = "Placeholder image for this popover modal optin form",
SIGNUP_TEXT = "Signup today for free and be the first to get notified on new updates.",
INPUT_PLACEHOLDER = "Enter your email",
SUBMIT_BUTTON = "Subscribe",
SUCCESS_MESSAGE = "Thanks for your subscription!",
ERROR_MESSAGE = "Submitting form failed!",
OPTIN =  true ,
COOKIE_NAME = "mycookie1",
CONTENTLANGUAGE = ""; 
</script>





<script type="text/javascript"
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js"></script>
<script type="text/javascript"
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/config/TeX-AMS-MML_HTMLorMML.js"></script>

<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    displayMath: [['$$','$$'], ['\[','\]']],
    processEscapes: true,
    processEnvironments: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
    TeX: { equationNumbers: { autoNumber: "AMS" },
         extensions: ["AMSmath.js", "AMSsymbols.js"] }
  }
});
</script>

<script type="text/x-mathjax-config">
  MathJax.Hub.Queue(function() {
    for(var all in MathJax.Hub.getAllJax()) {
        all.SourceElement().parentNode.className += ' has-jax';

    }
});
</script>



</head>
<body class="map[name:paper]" data-ng-app="myapp" data-ng-controller="MyController" data-ng-mouseleave="MouseLeave($event)">
    <header id="main-header">
  <nav class="navbar navbar-default navbar-fixed-top">
    <div class="container">
      <div class="navbar-header">
        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        
          
          <a class="navbar-brand-img" href="https://xinhe-lab.github.io/journal-club/">
            <img alt="UCHGCompBio" src="https://xinhe-lab.github.io/journal-club/img/jc-logo.png">
            
          </a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
          <ul class="nav navbar-nav navbar-right">
            
            
            <li class="">

              <a href="https://xinhe-lab.github.io/journal-club/post/" >
                <i class='fa fa-road'></i>
                Posts
              </a>
            </li>
            
            <li class="">

              <a href="https://github.com/xinhe-lab/journal-club" >
                <i class='fa fa-github'></i>
                Source
              </a>
            </li>
            
            
              
          </ul>
        </div>
        
      </div>
    </nav>
  </header>


<div class="container">
  <div class="row">
    <div class="col-sm-9">
      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
  <div class="text-center">

    <h1>Natural Variation in Gene Expression Modulates the Severity of Mutant Phenotypes
</h1>

    <div class="metas">
<small>
  <i class="fa fa-calendar"></i>
  <time datetime="2018-02-06">6 Feb, 2018</time>
</small>


  <small>
    &middot; by Yanyu Liang
  
  &middot; Read in about 3 min
  &middot; (616 words)
  &middot; 
<span class="share-box">Share this on:
    <a href="https://www.facebook.com/sharer/sharer.php?u=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-facebook-official "></i></a>

    <a href="https://twitter.com/intent/tweet?text=Natural%20Variation%20in%20Gene%20Expression%20Modulates%20the%20Severity%20of%20Mutant%20Phenotypes&amp;url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-twitter"></i></a>

    <a href="https://plus.google.com/share?url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-google-plus"></i></a>

    <a href="http://www.reddit.com/submit?url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=900,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-reddit"></i></a>

    <a href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f&amp;title=Natural%20Variation%20in%20Gene%20Expression%20Modulates%20the%20Severity%20of%20Mutant%20Phenotypes" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-linkedin"></i></a>

    <a href="mailto:?subject=Natural%20Variation%20in%20Gene%20Expression%20Modulates%20the%20Severity%20of%20Mutant%20Phenotypes&amp;body=Check out this site https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" data-proofer-ignore=""><i class="fa fa-envelope"></i></a>
  </span>

  </small>

<div class="margin-10">
  <i class="fa fa-tags"></i>
  
  <a href="https://xinhe-lab.github.io/journal-club/tags/epistasis" class="label label-primary">epistasis</a>
  


</div>

<br>
</div>

  </div>
</div>

      <div class="content">
  

<h2 id="biological-question">Biological question</h2>

<p>Mutation in monogenic disease can cause disease phenotype at various level of severity.</p>

<p>For instance, cystic fiberosis (CF) is caused by a single gene, <em>CFTR</em>, the homozygous individuals with exactly the same allele $\delta$F508 present a wide range of clinical symptoms. It is due to &laquo;modifier genes&raquo;, <em>MBL2</em> and <em>TGF-$\beta$</em>.</p>

<p>Such observation is general in many Mendelian diseases. Namely, the disease is predictable but the severity of it is highly unknown due to the variation of genetic background.</p>

<p>So, the biological question is: How does genetic background affect the severity of a mutation? (How ubiquitous? Is there underlying pattern?)</p>

<h2 id="main-finding">Main finding</h2>

<h2 id="experimental-approach">Experimental approach</h2>

<h3 id="gene-set">Gene set</h3>

<p>There are many known RNAi knockdown phenotypes in <em>C. elegans</em> (i.e. sterility, lethality, tissue development defect, etc). 1,353 genes were selected and their outcomes were tested on two <em>C. elegans</em> isolates (N2 Bristol and CB4856; two distinct genetic backgrounds; differ 1 SNP per 800 bp on average).</p>

<h3 id="quantifying-phenotype">Quantifying phenotype</h3>

<h4 id="fact">Fact</h4>

<p><em>C. elegans</em> life: L1 -&gt; L2 -&gt; L3 -&gt; adults</p>

<h4 id="procedure">Procedure</h4>

<p>15 L1 worms + RNAi -&gt; -&gt; 4-day observation -&gt; -&gt; 15 adults with their L1, L2, L3 progeny. The outcome is compared with respect to negative control to negative control to be quantified.</p>

<h4 id="measurement">Measurement</h4>

<p><img src="https://xinhe-lab.github.io/journal-club/victoria-2015-cell/fig1.png" alt="Figure 1" /></p>

<h4 id="correcting-for-rnai-uptake">Correcting for RNAi uptake</h4>

<p>RNAi has different efficacy in N2 and CB4856 due to null mutation in <em>ppw-1</em> in CB4856. It has been corrected for cases where N2 has a more severe phenotype in N2.</p>

<h4 id="false-positives-of-rnai-approach">False positives of RNAi approach</h4>

<p>8 genes with differential severity in RNAi experiment were confirmed by genetic mutants. 6 out of 8 genes were detected with the same effect.</p>

<h2 id="results">Results</h2>

<h3 id="functional-enrichment">Functional enrichment</h3>

<p>For genes causing more severe phenotypes in N2/CB4856, whether they tend to have similar function? For instance, genes affecting N2 enriched in electron transport chain (ETC) and P granule regulator whereas genes affecting CB4856 enriched in protein synthesis and fibrous organelle (FO). The same sensitivity trend was found using chemical to interfere the worm.</p>

<p><img src="https://xinhe-lab.github.io/journal-club/victoria-2015-cell/fig4.png" alt="Figure 4" /></p>

<h3 id="variation-in-gene-expression">Variation in gene expression</h3>

<h4 id="functional-module-activity-severity">Functional module activity ~ severity</h4>

<p>20% genes have differential severity but the coding sequences are almost the same between two worms, so that they hypothesized that the difference is due to expression level difference of the gene.</p>

<p>For the four functional modules which enrich genes with the same direction, the expression level of the proteins in the module (module baseline activity) is consistent with the severity result (high severity -&gt; high sensitivity -&gt; low baseline activity)</p>

<p><img src="https://xinhe-lab.github.io/journal-club/victoria-2015-cell/fig5a.png" alt="Figure 5A (note that the RNA-seq was done in wildtype w/o any treatment)" /></p>

<h4 id="gene-baseline-expression-severity">Gene baseline expression ~ severity</h4>

<p>In particular, for the detected genes (which is a member of the module), its baseline expression level in two worms consistent with their severity as well.</p>

<h4 id="altering-gene-baseline-expression-severity">Altering gene baseline expression -?-&gt; severity</h4>

<p><em>vab-10</em> is in FO module which has high activity in N2. N2 strain with <em>vab-10</em> homozygous hypomorphic allele decreases the baseline activity of <em>vab-10</em>.</p>

<p><img src="https://xinhe-lab.github.io/journal-club/victoria-2015-cell/fig5b.png" alt="Figure 5B" /></p>

<h4 id="altering-functional-module-activity-severity">Altering functional module activity -?-&gt; severity</h4>

<p><em>unc-52</em> is also in FO module.</p>

<p><img src="https://xinhe-lab.github.io/journal-club/victoria-2015-cell/fig5c.png" alt="Figure 5C" /></p>

<h4 id="replication-in-cb4856">Replication in CB4856</h4>

<p><em>cgh-1</em> is in P granule module which has high activity in CB4856. Also, <em>pos-1</em> and <em>mex-3</em> are in P granule module.</p>

<p><img src="https://xinhe-lab.github.io/journal-club/victoria-2015-cell/fig5d.png" alt="Figure 5D" /></p>

<p>The effect is more than additive.</p>

<h3 id="gene-expression-is-predictive-for-severity">Gene expression is predictive for severity</h3>

<p>They explored the relationship between gene expression and severity by adding two more <em>C. elegans</em> isolates, AB1 and ED3040. They focused on the ETC functional module. (A: gene expression level; B: severity in each worm type; C: severity against expression level (relative to species average); D: sensitivity to chemical; E: correlation of expression and severity in mammalian cell (18 cell lines were used); F: one instance of E)</p>

<p><img src="https://xinhe-lab.github.io/journal-club/victoria-2015-cell/fig6.png" alt="Figure 6" /></p>

<h2 id="discussion">Discussion</h2>

<p>Genetic background impacts the severity of the genetic perturbation. In particular, natural variation of gene expression impacts the severity of mutation. In other words, the effect size of mutation depends on the genetic background.</p>

</div>


      <footer>
  
<span class="share-box">Share this on:
    <a href="https://www.facebook.com/sharer/sharer.php?u=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-facebook-official "></i></a>

    <a href="https://twitter.com/intent/tweet?text=Natural%20Variation%20in%20Gene%20Expression%20Modulates%20the%20Severity%20of%20Mutant%20Phenotypes&amp;url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-twitter"></i></a>

    <a href="https://plus.google.com/share?url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-google-plus"></i></a>

    <a href="http://www.reddit.com/submit?url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=900,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-reddit"></i></a>

    <a href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f&amp;title=Natural%20Variation%20in%20Gene%20Expression%20Modulates%20the%20Severity%20of%20Mutant%20Phenotypes" onclick="window.open(this.href, 'mywin',
'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"><i class="fa fa-linkedin"></i></a>

    <a href="mailto:?subject=Natural%20Variation%20in%20Gene%20Expression%20Modulates%20the%20Severity%20of%20Mutant%20Phenotypes&amp;body=Check out this site https%3a%2f%2fxinhe-lab.github.io%2fjournal-club%2f20180206-victoria-2015-cell%2f" data-proofer-ignore=""><i class="fa fa-envelope"></i></a>
  </span>

  <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
      
  
    <nav><ul class="pager">
    
        <li class="previous">
          <a href="https://xinhe-lab.github.io/journal-club/20171205-burgess-2016-nrg/" title="Regulatory elements: Putting enhancers into context">
            <span aria-hidden="true">&larr;</span>Previous
          </a>
        </li>
    

    
      <li class="next">
        <a href="https://xinhe-lab.github.io/journal-club/20180220-reddy-2012-gr/" title="Effects of sequence variation on differential allelic transcription factor occupancy and gene expression">
            Next <span aria-hidden="true">&rarr;</span>
        </a>
      </li>
    
    </ul> </nav>
  


</div>

  <div id="gh-comments">
    <br/><br/>
    <h4>COMMENTS</h4>
    <div id="gh-comments-list"></div>
    <a href="javascript:void(0)" id="gh-load-comments" class="btn" style="display:none">Load more comments</a>
</div>

<script type="text/javascript" src="https://xinhe-lab.github.io/journal-club/js/github-comments.js"></script>
<script type="text/javascript">
    DoGithubComments( 5 );
</script>

</footer>

    </div>
    
      <div class="col-xs-12 col-sm-12 col-md-3 col-lg-3">
        <div>
  

    <div class="section">
      <header><div class="title"><b>Latest Posts</b></div></header>
      <div class="content">
        <ul>
        
          <li>
          <a href="https://xinhe-lab.github.io/journal-club/20180220-reddy-2012-gr/">Effects of sequence variation on differential allelic transcription factor occupancy and gene expression</a>
          </li>
        
          <li>
          <a href="https://xinhe-lab.github.io/journal-club/20180206-victoria-2015-cell/">Natural Variation in Gene Expression Modulates the Severity of Mutant Phenotypes</a>
          </li>
        
          <li>
          <a href="https://xinhe-lab.github.io/journal-club/20171205-burgess-2016-nrg/">Regulatory elements: Putting enhancers into context</a>
          </li>
        
          <li>
          <a href="https://xinhe-lab.github.io/journal-club/20171205-seurat-2015-nbt/">Spatial reconstruction of single-cell gene expression data</a>
          </li>
        
          <li>
          <a href="https://xinhe-lab.github.io/journal-club/20171129-uchg-compbio-jc/">Read-n-share</a>
          </li>
        
        </ul>
      </div>
    </div>

    
      
      
    
      
      
      <div class="section taxonomies">
        <header><div class="title"><b>tag</b></div></header>

        <div class="content">
          <ul>
            <li><a href="https://xinhe-lab.github.io/journal-club/tags/regulatory-sequences">regulatory-sequences</a></li><li><a href="https://xinhe-lab.github.io/journal-club/tags/epistasis">epistasis</a></li><li><a href="https://xinhe-lab.github.io/journal-club/tags/single-cell-rna-seq">single-cell-rna-seq</a></li>
          </ul>
        </div>
      </div>
      
    
      
      
      <div class="section taxonomies">
        <header><div class="title"><b>topic</b></div></header>

        <div class="content">
          <ul>
            <li><a href="https://xinhe-lab.github.io/journal-club/topics/regulatory-genomics">regulatory-genomics</a></li><li><a href="https://xinhe-lab.github.io/journal-club/topics/genetics">genetics</a></li><li><a href="https://xinhe-lab.github.io/journal-club/topics/single-cell">single-cell</a></li>
          </ul>
        </div>
      </div>
      
    

</div>

      </div>
    
  </div>
</div>
    
<footer class="footer hidden-print">
  <div class="container">
    <div class="row">
        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
           <div class="pull-left">
  <a class="toplink" href="javascript:" id="return-to-top">back to top</a>
</div>
<div class="pull-right">

</div>

        </div>
        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 text-center">
              
    

    
<div class="container copyright">
    <small>
  &copy; 2017 Copyright Human Genetics Computational Group | The University of Chicago

  </small>
</div>



        </div>
    </div>
  </div>
</footer>

  </body>
</html>

